### DISCUSSION TITLE
WordMat menuen mangler i Word after en successful deployment fra Intune.

### DISCUSSION THREAD

**Discussion by IndYank310** [Eduap-com/WordMat#233] [Support]
Versionen af WordMat i drift hos os er 1.24.1. Den virker mere eller mindre OK. 
Installationen af den sker fra MS Intune hvor app’en er pakket som en Win32App som installerer i System Context. Jeg har forsøgt nu at opgradere til Version 1.30. 
Min installations parameter er: /SP /VERYSILENT /SUPPRESSMSGBOXES /ALLUSERS /LOG="C:\ITControl\wordmat130.log" /NORESTART /MERGETASKS="!desktopicon" /COMPONENTS="Graph,!Geogebra"

Installationen på test miljø er successful. Log filen viser = ingen fejl. Men,
**Når jeg åbner Word, Wordmat menuen mangler.** 

Når jeg forsøger en **manual commandline installation** på en test pc, med de samme installationsparametre, bliver applikationen installeret fint og Wordmat menuen findes i Word. 

Er der noget jeg laver forkert? :-)

Office version 2406 Office 365.
OS = Windows 10 Education 22H2

**Comment by IndYank310** [comment]
Et par ord til: jeg fik nu lavet en ekstra test. jeg fik konfigureret en ny Intune pakke som er præcis det samme som den anden pakke med kun 1 forskel - den instalerer i user context. Installations test på maskiner hvor brugeren er en **lokal administrator** installeret succefuld og Wordmat menuen var til stede. Installation på en maskine hvor brugeren er en **almindelig bruger** failed. Årsagen til at den mislykkedes fandt jeg i Intune extension log = "error code -2147024895' The system cannot find the path specified." En af årsage til denne slags fejl er en user tilladelser (User Permissions) issue. Er der en måde at installere Wordmat 1.30 og få den til at virke på maskiner hvor brugeren er en almindelig bruger?

**Reply by Oestrup** [reply]
Jeg har ikke kunne få Wordmat menuen frem efter en deployment siden Wordmat version 1.25.2. 
Prøv at installere Wordmat version 1.25.2 og se om ikke menuen kommer frem i den version. Det gør den hos mig. 
Jeg har ikke kunne knække koden for de senere versioner.

**Reply by IndYank310** [reply]
Hej, og mange tak for dit svar. Jeg er temmelig sikker på at årsagen ligger i en fejl i Wordmat fra version 1.24.x (som fungerer hos os) og senere. Jeg ser det som en alvorlig fejl fordi, i mine øjne, den gør WordMat uegnet til proffesionel brug. Jeg gætter på at programmet prøver at skrive til en bruger dedikerert folder eller til CurrentUser hive. Når den bliver installeret i system kontext, skriver den i systemets konto og derfor 'mangler' menuen hos brugerne. WordMat kræver nemlig minimum lokal admin rettigheder for at opnå succesful installtion som CurrentUser.  Anyway, jeg har meldt ud herind, at indtil videre, opdateres WordmMat ikke hos os. Jeg håber at fejlen vil bliver rettet i fremtidige versioner.

**Reply by Oestrup** [reply]
Vi har så den mulighed at vi kan lade vores brugere installere programmer med lokal admin rettigheder. Det er langt fra en optimalt måde at distribuere programmer ud på. 
Problemet er bare efter version 1-25-2, blokere vores antivirusprogram for WordMat.dotm. Så vi er lige vidt. 
Der bliver heller ikke opgraderet Wordmat hos os, før disse problemer bliver løst.

**Reply by bjarneec85** [reply]
Jeg har løst det ved at lave et script der kopiere dotm filerne fra WordMat mappen over i roaming/Microsoft/Word mappen. Det lader til at løse det.

**Reply by InterceptorDK** [reply]
Siden version 27 har jeg ikke kunnet installere det via deployment, hvor det er System kontoen der installerer. 
Det er ikke lykkedes mig at finde en workaround endnu, vi er heller ikke admins.

**Reply by Oestrup** [reply]
Jeg kan godt lave en deployment, og som @bjarneec85 foreslog med at lave et script med at kopiere filerne fra Wordmat mappen og i roaming/Microsoft/Word virker også....MEN, Withsecure som er vores antivirus program registere at der er Malware.W2000M/AVI.snh.rqgah i WordMat.dotm og sætter den i karantæne. Så jeg er slået tilbage til start og har desværre ikke kunne løse at få fjernet den karantæne regel.
